app/models/name.rb

```diff
@@ -2219,12 +2102,64 @@ def self.new_name(params)
     result
   end
 
+  # Make a Name instance from a ParsedName
+  # Used by NameController#create_new_name
+  # Returns a Name instance, *UNSAVED*!!
+  def self.new_name_from_parsed_name(parsed_name)
+    new_name(parsed_name.params)
+  end
+
+  # Return extant Names matching a desired new Name
+  # Used by NameController#create_name
+  def self.names_matching_desired_new_name(parsed_name)
+    # unauthored ParsedName matches Names with or w/o authors
+    if parsed_name.author.empty?
```

Comment:
I haven't actually tested this, but it looks like it might be impossible to create "Blah group" if "Blah group sensu Author" already exists.  Maybe just moving the "if parsed_name.rank == :Group" to the top of the conditional would work?  I think the behavior is correct for other ranks.  That is, if "Genus Author" already exists, then it shouldn't let you create "Genus" without author.  I realize that you're allowed to modify an existing name to not have an author.  And that's fine by me.  But it smells funny that you have to create "Blah group Bogus Author" first, then edit the name to remove "Bogus Author". ...  Or you've already foreseen this case and I'm just not following you right!  In that case just ignore me.